client/plots/violin.renderer.js

```diff
@@ -429,7 +429,6 @@ function getLegendGrps(self) {
 		t1 = self.config.term,
 		t2 = self.config.term2,
 		headingStyle = 'color: #aaa; font-weight: 400'
-	let title, name
```

Comment:
@karishma-gangwani Please avoid sharing variables between functions or loops, unless those variables are really meant to be shared state. In this case, the title/name is unique to a legend entry, there is no reason to share that variable value.